src/Pos/Update/Arbitrary/Core.hs

```diff
@@ -52,7 +52,7 @@ instance Arbitrary UpdateProposal where
                   upSoftwareVersion
                   upData
                   upAttributes
-        pure UpdateProposal {..}
+        pure UnsafeUpdateProposal {..}
```

Comment:
Oh, excuse me, I am wrong about signing logic, because `mkUpdateProposal` doesn't contain signing logic. I think it makes sense to have two constructor: one taking existing signature, another one taking secret key and signing data internally.